Backend/tests/mock_data_2.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "email" : "earth@mentii.me",
+    "password" : "iameight",
+    "activationId" : "12345",
+    "active": "T"
+  },
+  {
+    "email" : "wind@mentii.me",
```

Comment:
i actually was thinking of earth, wind, fire the band and then thought of avatar the last airbender which had air last